avogadro/rendering/mesh_opaque_fs.glsl

```diff
@@ -1,4 +1,10 @@
-varying vec3 fnormal;
+#version 430
```

Comment:
Do we need to go to version 430? I thought the general advice was to use the lowest version of GLSL that works for your needed code / features (e.g., we should use `#version 400`)